contracts/airdrop/src/contract.rs

```diff
@@ -301,11 +348,6 @@ pub fn handle_delegate_astro_to_bootstrap_auction(
     let mut state = STATE.load(deps.storage)?;
     let mut user_info = USERS.load(deps.storage, &info.sender)?;
 
-    // CHECK :: HAS USER ALREADY WITHDRAWN THEIR REWARDS ?
```

Comment:
Do we allow users to withdraw the rewards as many time as needed?

Reply:
not needed imo, this airdrop is one-time only, so better to have user call the `WithdrawAirdropReward` / `Claim` (after auction) fn once and we send him all his withdrawable ASTRO airdrop tokens